ilastik/applets/tracking/base/trackingBaseDataExportApplet.py

```diff
@@ -185,4 +203,79 @@ def _configure_operator_with_parsed_args(cls, parsed_args, opTrackingDataExport)
         # configure super operator
         DataExportApplet._configure_operator_with_parsed_args(parsed_args, opTrackingDataExport)
 
+    @property
+    def op(self):
```

Comment:
any compelling reason not to use the `self.toplevelOperator` property, implemented in `98` of this file?